docs/02.API-REFERENCE.md

```diff
@@ -3868,6 +3868,114 @@ example (void)
 - [jerry_create_promise](#jerry_create_promise)
 - [jerry_promise_state_t](#jerry_promise_state_t)
 
+## jerry_from_property_descriptor
+
+**Summary**
+
+This API function is equivalent to FromPropertyDescriptor operation.
+Based on ECMA 262 v11 6.2.5.4 this is in the form of `FromPropertyDescriptor`.
```

Comment:
This API function is equivalent to FromPropertyDescriptor operation defined in ECMA 262 v11 6.2.5.4. It returns with an ECMAScript Object which represents the property attributes.